website/templates/project/view_file.mako

```diff
@@ -1,5 +1,16 @@
 <%inherit file="project/project_base.mako"/>
 <%def name="title()">${file_name | h}</%def>
+
+% if (user['can_comment'] or node['has_comments']):
+    % if allow_comment:
+        <%include file="include/comment_pane_template.mako"/>
+    % else:
+        <div class="alert alert-warning" role="alert">
+            Comments for this addon is not yet supported.
+        </div>
```

Comment:
Right, since comments are not allowed for dataverse files, I think those file pages should include the warning alert:
```<div class="alert alert-warning" role="alert">
           Comments for this addon is not yet supported.
    </div>```